Sources/PaymentsCore/Networking/APIClient.swift

```diff
@@ -46,4 +46,20 @@ public class APIClient {
         let (response, _) = try await fetch(endpoint: request)
         return response.clientID
     }
+    
+    /// :nodoc: This method is exposed for internal PayPal use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    /// - Parameter name: Event name string used to identify this unique event in FPTI.
+    public func sendAnalyticsEvent(name: String) async {
```

Comment:
On Braintree we do the parameter for name as an anonymous parameter so it the call site we can do `client.sendAnalyticsEvent("my-super.cool.event")`. Do we prefer that over what is here `client.sendAnalyticsEvent(name: "my-super.cool.event")`. I think it reads a little cleaner but curious how y'all feel!

Reply:
I like this! Fixed